bin/index.js

```diff
@@ -1,15 +1,20 @@
 #! /usr/bin/env node
-const got = require('got')
 
-const { backupMessages } = require("./backupMessages");
+const { defaultCommand } = require("./commands");
 
-const generateRandomNumber = Math.floor(Math.random() * (backupMessages.length - 1));
+const args = {
+  PREFIX: 'prefix'
+};
 
-(async () => {
-	try {
-		const response = await got('http://whatthecommit.com/index.txt');
-		console.log(response.body);
-	} catch (error) {
-		console.log(backupMessages[generateRandomNumber]);
-	}
-})();
\ No newline at end of file
+const firstUserArg = process.argv.slice(2)[0];
+
+switch (firstUserArg) {
+  case args.PREFIX:
+    defaultCommand({ prefix: true });
```

Comment:
```suggestion
    defaultCommand({ prefix });
```
sugar: Is it possible to do this?

Reply:
not going to address